Parse/Internal/File/State/FileState.cs

```diff
@@ -1,11 +1,31 @@
-// Copyright (c) 2015-present, Parse, LLC.  All rights reserved.  This source code is licensed under the BSD-style license found in the LICENSE file in the root directory of this source tree.  An additional grant of patent rights can be found in the PATENTS file in the same directory.
-
-using System;
-
-namespace Parse.Internal {
-  internal class FileState {
-    public string Name { get; internal set; }
-    public string MimeType { get; internal set; }
-    public Uri Url { get; internal set; }
-  }
-}
+// Copyright (c) 2015-present, Parse, LLC.  All rights reserved.  This source code is licensed under the BSD-style license found in the LICENSE file in the root directory of this source tree.  An additional grant of patent rights can be found in the PATENTS file in the same directory.

```

Comment:
Newline format got rewritten here. We should setup a project wide format for this and just use it, having files be rewritten is getting annoying.